test/e2e/storage/testsuites/base.go

```diff
@@ -61,14 +61,15 @@ type TestSuite interface {
 	// defineTest defines tests of the testpattern for the driver.
 	// Called inside a Ginkgo context that reflects the current driver and test pattern,
 	// so the test suite can define tests directly with ginkgo.It.
-	defineTests(TestDriver, testpatterns.TestPattern)
+	defineTests(TestDriver, testpatterns.TestPattern, volume.SizeRange)
 }
 
 // TestSuiteInfo represents a set of parameters for TestSuite
 type TestSuiteInfo struct {
-	name         string                     // name of the TestSuite
-	featureTag   string                     // featureTag for the TestSuite
-	testPatterns []testpatterns.TestPattern // Slice of TestPattern for the TestSuite
+	name               string                     // name of the TestSuite
+	featureTag         string                     // featureTag for the TestSuite
+	testPatterns       []testpatterns.TestPattern // Slice of TestPattern for the TestSuite
+	supportedSizeRange volume.SizeRange           // Size range supported by the test suite
```

Comment:
Specifying a maximum size is necessary if It use `getSizeRangesIntersection`  otherwise it may skip because it don't know how to decide `Intersection`